src/fhirtypes/InstanceDefinition.ts

```diff
@@ -10,8 +11,8 @@ import { Meta } from './specialTypes';
 export class InstanceDefinition {
   resourceType: string;
   instanceName: string;
-  id?: string;
   meta?: Meta;
+  id?: FHIRId;
```

Comment:
Is there a reason you swapped the order here?

Reply:
Nope, that was just me moving quickly when resolving rebase conflicts. I'll put it back.